libr/core/cmd_type.c

```diff
@@ -824,7 +826,7 @@ static int cmd_type(void *data, const char *input) {
 		case 'b': // "teb"
 			res = r_type_enum_member (TDB, name, member_name, 0);
 			break;
-		case 'c' : { // "tec"
+		case 'c' : {
```

Comment:
Don't remove these comments, since they are required for searching the command implementation.

Reply:
Done!